service/test/NVMLIOGroupTest.cpp

```diff
@@ -346,7 +359,7 @@ TEST_F(NVMLIOGroupTest, read_signal)
         // FIXME: The most complex signal is the cpu accelerator active afifinitzation signal, which is currently
```

Comment:
Do we want to include the fix in this specific pull request or in another one?

Reply:
This is outside the scope of this pull request.